### PR TITLE
Increase resize file limit to allow for HEIC files #174182278

### PR DESCRIPTION
**Pull request by akegan**
It turns out that the HEIC file was failing at the [resize step](https://github.com/SFDigitalServices/sf-dahlia-web/pull/1404/files#diff-4a17de4b00d35126ebac5a0d036ab3c8L82). 

I've just updated that line to not try to resize HEIC files

Review instructions in [the ticket](https://www.pivotaltracker.com/story/show/174182278).

### CODE REVIEW

**Comment by akegan** [comment]
@james-wills-sf can you QA on your phone too to make sure that this is going to work there too?

File: app/assets/javascripts/short-form/components/file-uploader.html.slim [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
   / The ngf-max-size attr is set to 20MB, which is different from the 5MB limit in FileUploadService, because we try to resize the file in  _processProofFile to see if we can get it under 5MB
-  button.button.secondary ng-class="{error: $ctrl.inputInvalid($ctrl.fileInputName), 'ghost-white': !$ctrl.inputInvalid($ctrl.fileInputName)}" type="button" ngf-select="$ctrl.uploadFile($file, $ctrl.opts)" ngf-accept="'image/jpg, image/jpeg, image/png, application/pdf'" name="{{$ctrl.fileInputName}}" ng-required="$ctrl.required" aria-describedby="{{$ctrl.fileInputName}}_error" ng-model="$ctrl.document.file" id="{{$ctrl.fileInputName}}" ngf-validate-fn="$ctrl.validateFile($file)"
+  button.button.secondary[
+      ng-class="{error: $ctrl.inputInvalid($ctrl.fileInputName), 'ghost-white': !$ctrl.inputInvalid($ctrl.fileInputName)}"
+      type="button"
+      id="{{$ctrl.fileInputName}}"
+      name="{{$ctrl.fileInputName}}"
+      aria-describedby="{{$ctrl.fileInputName}}_error"
+      ng-model="$ctrl.document.file"
+      ng-required="$ctrl.required"
```
Comment (akegan) [inline app/assets/javascripts/short-form/components/file-uploader.html.slim:11]: could not tell you why ngf-pattern and ngf-max-size work on pref proof uploads but not this one. I am not sure why the 2 are different but am tempted to leave it as is

File: app/assets/javascripts/short-form/components/file-uploader.html.slim [hunk 2]
```diff
@@ -1,7 +1,18 @@
 .form-group.text-center ng-class="{ error: $ctrl.inputInvalid($ctrl.fileInputName) }" ng-hide="$ctrl.hasFile()"
   / type="button" tells it not to act as a submit button
   / The ngf-max-size attr is set to 20MB, which is different from the 5MB limit in FileUploadService, because we try to resize the file in  _processProofFile to see if we can get it under 5MB
-  button.button.secondary ng-class="{error: $ctrl.inputInvalid($ctrl.fileInputName), 'ghost-white': !$ctrl.inputInvalid($ctrl.fileInputName)}" type="button" ngf-select="$ctrl.uploadFile($file, $ctrl.opts)" ngf-accept="'image/jpg, image/jpeg, image/png, application/pdf'" name="{{$ctrl.fileInputName}}" ng-required="$ctrl.required" aria-describedby="{{$ctrl.fileInputName}}_error" ng-model="$ctrl.document.file" id="{{$ctrl.fileInputName}}" ngf-validate-fn="$ctrl.validateFile($file)"
+  button.button.secondary[
```
Comment (james-wills-sf) [inline app/assets/javascripts/short-form/components/file-uploader.html.slim:4]: good to know you can wrap attributes using `[`!

File: app/assets/javascripts/short-form/components/file-uploader.html.slim (hunk 1 above)
Comment (james-wills-sf) [reply app/assets/javascripts/short-form/components/file-uploader.html.slim:11]: I know how to test the preference proof upload, but where is this element in the application flow?

Comment (akegan) [reply app/assets/javascripts/short-form/components/file-uploader.html.slim:11]: It's the first page of the Sale application where people have to upload proof of homebuyer education

**Comment by james-wills-sf** [comment]
@akegan Just tested uploading on a sale listing, it works!

**Pull request merged**